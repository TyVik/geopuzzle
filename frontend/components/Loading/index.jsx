'use strict';
import React from "react";

import './index.css'
import {FormattedMessage as Msg} from "react-intl";


const Loading = (props) => {
  return <div className="loading_wrapper">
    <h2 className="loading">
      <Msg id={props.hasError ? "loadingError": "loading"} />
    </h2>
  </div>;
};


export default Loading;
