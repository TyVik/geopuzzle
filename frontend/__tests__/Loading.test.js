'use strict';
import React from 'react';
import {renderWithIntl, loadTranslation} from 'enzyme-react-intl';
import Loading from '../components/Loading';

loadTranslation("./frontend/locale/en.json");


it('shallow <Loading /> components', () => {
  expect(renderWithIntl(<Loading text='test'/>)).toMatchSnapshot();
});
