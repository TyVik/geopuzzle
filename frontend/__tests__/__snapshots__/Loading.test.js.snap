// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow <Loading /> components 1`] = `
<div
  class="loading_wrapper"
>
  <h2
    class="loading"
  >
    <span>
      Loading...
    </span>
  </h2>
</div>
`;
